x-pack/plugin/rank-rrf/src/test/java/org/elasticsearch/xpack/rank/rrf/RRFRetrieverBuilderParsingTests.java

```diff
@@ -111,16 +115,25 @@ public void testRRFRetrieverParsing() throws IOException {
                   "retrievers": [
                     {
                       "test": {
-                        "value": "foo"
+                        "value": "foobar"
```

Comment:
Let's not modify this existing test, but add new ones?

You could have a separate private method e.g. `checkRRFRetrieverParsing` that takes in as an argument `restContent` and encapsulates the test logic we have here:

https://github.com/elastic/elasticsearch/blob/75ca87436bebb5aec68e2495ed7acc4035ab085a/x-pack/plugin/rank-rrf/src/test/java/org/elasticsearch/xpack/rank/rrf/RRFRetrieverBuilderParsingTests.java#L133-L150

and then we can have different test methods that call `checkRRFRetrieverParsing` with different strings representing different formats of the RRF retriever that we need to test.